src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -114,7 +114,7 @@ private function validate($content, $file = null)
         try {
             $this->getParser()->parse($content, Yaml::PARSE_CONSTANT);
         } catch (ParseException $e) {
-            return array('file' => $file, 'valid' => false, 'message' => $e->getMessage());
+            return array('file' => $file, 'line' => $e->getParsedLine(), 'valid' => false, 'message' => $e->getMessage());
```

Comment:
Yes I've noticed that while working with this. I want to research it a little more because the message will have a line number sometimes, so I will try to find out why it is not returning for `getParsedLine()`.